dataland-keycloak/dataland_theme/email/text/password-reset.ftl

```diff
@@ -2,7 +2,7 @@
 Hi,
 
```

Comment:
Das text-file der verification mail müsste auch noch angepasst werden.
Bzw. darin wird auf die property "emailVerificationBody" verwiesen. D.h. wir müssten diese in die theme.properties hier aufnehmen und customizen.
Keycloak erkennt dann, dass wir einen eigenen Wert für "emailVerificationBody" haben, und überschreibt den default (der aktuell genutzt wird).

Reply:
Hier kann man den default Wert abchecken:
https://github.com/keycloak/keycloak/blob/cdef39f76c50506681227b627e678bdbf15779a4/themes/src/main/resources/theme/base/email/messages/messages_en.properties

Reply:
fixed